packages/web-sdk/src/instrumentations/performance/instrumentation.ts

```diff
@@ -0,0 +1,32 @@
+import { BaseInstrumentation, VERSION } from '@grafana/faro-core';
+
+import { observeAndGetNavigationTimings } from './navigation';
+import { onDocumentReady, performanceObserverSupported } from './performanceUtils';
+import { observeResourceTimings } from './resource';
+
+export class PerformanceInstrumentation extends BaseInstrumentation {
+  readonly name = '@grafana/faro-web-sdk:instrumentation-performance';
+  readonly version = VERSION;
+
+  initialize() {
```

Comment:
As a side note, I suggest that we provide a way for the consumers of the instrumentation(s) to pause/resume the instrumentation. The way this works for now, it will only stop doing its thing after a refresh

Reply:
Sounds good!